src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -48,29 +36,26 @@ protected function configureRoutes(RouteCollectionBuilder $routes)
      *         ->controller('App\Controller\AdminController::dashboard')
      *     ;
      */
-    protected function configureRouting(RoutingConfigurator $routes): void
-    {
-        @trigger_error(sprintf('Not overriding the "%s()" method is deprecated since Symfony 5.1 and will trigger a fatal error in 6.0.', __METHOD__), E_USER_DEPRECATED);
-    }
+    abstract protected function configureRoutes(RoutingConfigurator $routes);
```

Comment:
Changing the typehint with the existing method name is a BC break.

Reply:
it's not a BC break because this is a trait: everyone is forced to override it right now.